onnxruntime/core/graph/contrib_ops/reverse_sequence_schema_defs.cc

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "reverse_sequence_schema_defs.h"
+#include "core/graph/constants.h"
+#include "core/graph/op.h"
+#include <cmath>
+#include <type_traits>
+
+namespace onnxruntime {
+namespace contrib {
+
+using ::ONNX_NAMESPACE::AttributeProto;
+using ::ONNX_NAMESPACE::OPTIONAL;
+using ::ONNX_NAMESPACE::OpSchema;
+using ::ONNX_NAMESPACE::InferenceContext;
+using ::ONNX_NAMESPACE::TensorShapeProto;
+using ::ONNX_NAMESPACE::TensorProto;
+using ::ONNX_NAMESPACE::TensorProto_DataType;
+
+static const char* ReverseSequence_ver1_doc = R"DOC(
+Reverse batch of sequences having different lengths specified by `sequence_lens`.
+
+For each slice i iterating on batch axis, the operator reverses the first sequence_lens[i] elements on time axis.
+
+Example 1:
+  input = [
+    [0.0, 1.0, 2.0],
+    [3.0, 4.0, 5.0],
+  ]
+  sequence_lens = [3, 3]
+  data_format = "batch_major"
+
+  output = [
+    [2.0, 1.0, 0.0],
+    [5.0, 4.0, 3.0],
+  ]
+
```

Comment:
I don't think this example adds any value given the sequence length doesn't cause any truncation.